sdk/storage/Azure.Storage.Webjobs.Extensions.Common/src/Shared/Constants.cs

```diff
@@ -5,11 +5,6 @@ namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common
 {
     internal static class Constants
     {
-        public const string WebJobsConfigurationSectionName = "AzureWebJobs";
-        public const string EnvironmentSettingName = "AzureWebJobsEnv";
-        public const string DevelopmentEnvironmentValue = "Development";
-        public const string DynamicSku = "Dynamic";
-        public const string AzureWebsiteSku = "WEBSITE_SKU";
```

Comment:
where did this go?

Reply:
Wasn't used

Reply:
Ah, these were copy pasted elsewhere at some point.